test/test_meta.py

```diff
@@ -1602,6 +1602,66 @@ def test_quantized_embedding_bag(self):
         self.assertEqual(eb.dtype, torch.float32)
         self.assertEqual(eb.untyped_storage().data_ptr(), 0)
 
+    # Tests mean and max.
+    # Can't easily test sum, because there is a fast path for sum which
+    # causes offset2bag to not get allocated... but the backward function
+    # needs it, and the offset2bag computation lives inside the
+    # derivatives.yaml formula directly, so there is no way to access it.
```

Comment:
I'm not sure what you mean that the computation lives in the formula?

Reply:
you can have derivatives.yaml derivative do some compute, e.g., by directly calling a C++ function, before it eventually goes to an actual dispatched op